Shoot/Shoot/ViewController.swift

```diff
@@ -176,49 +176,37 @@ class ViewController: UIViewController, UIImagePickerControllerDelegate, UINavig
     // MARK - UIImagePickerControllerDelegate
 
     func imagePickerController(_ picker: UIImagePickerController, didFinishPickingMediaWithInfo info: [String: Any]) {
-        self.dismiss(animated: true, completion:nil)
+        self.dismiss(animated: true, completion: nil)
         let image: UIImage = info[UIImagePickerControllerOriginalImage] as! UIImage
         if (newMedia == true) {
             UIImageWriteToSavedPhotosAlbum(image, self, #selector(ViewController.image(_:didFinishSavingWithError:contextInfo:)), nil)
         } else {
-            let imageURL:URL = info[UIImagePickerControllerReferenceURL] as! URL
-            let assetslibrary = ALAssetsLibrary()
-            
-            assetslibrary.asset(for: imageURL, resultBlock: {
-                (asset: ALAsset?) in
-                if let asset = asset {
-                    let assetRep: ALAssetRepresentation = asset.defaultRepresentation()
-                    self.imageView.accessibilityIdentifier = assetRep.filename()
```

Comment:
turns out this code uses ALAssetsLibrary for getting the file name, so just removing the deprecated code makes the publish crash as there is no file name. So we have to find a way of getting the file name.

Reply:
Hrm, @jcesarmobile I didn't get any crash. Which service have you tested?